app/javascript/components/admin/UserModal.js

```diff
@@ -131,6 +131,22 @@ class UserModal extends React.Component {
                 onChange={this.handleAPIAccessChange}
               />
             </Form.Group>
+            <Form.Group>
+              <Form.Label>Notes</Form.Label>
+              <InputGroup>
+                <Form.Control
+                  id="notes"
+                  name="notes"
+                  as="textarea"
+                  rows="5"
+                  className="form-square"
+                  defaultValue={this.state.notes}
```

Comment:
this should be `value` not `defaultValue`

Reply:
Fixed in https://github.com/SaraAlert/SaraAlert/pull/805/commits/b02f3a00163c7a98b7c07659635b1495dcdc5c78